frontend/src/utils/string-utils.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-bitwise, fp/no-mutation */
+
+// https://stackoverflow.com/a/7616484
+export const stringHash = (s: string): string => {
```

Comment:
nice, except that disabling two es-lint rules is a yellow flag :)

Maybe we could use something like this one-liner?

```
const hashCode = (str) => [...str].reduce((s, c) => Math.imul(31, s) + c.charCodeAt(0) | 0, 0) 
```

See https://gist.github.com/hyamamoto/fd435505d29ebfa3d9716fd2be8d42f0

Reply:
ok